docs/how-to-run.adoc

```diff
@@ -254,95 +342,87 @@ $ zoneadm list -cnv
 $ pfexec tail -f $(pfexec svcs -z oxz_nexus -L nexus)
 ----
 
-=== Uninstalling
+=== Finish configuring external networking
 
-To uninstall all Omicron services from a machine, the following may be
-executed:
+After installing omicron with `omicron-package install`, you can run the `softnpu-init.sh` script to configure SoftNPU.  If your external network is the one that your global zone is already on (i.e., an existing network), then you can likely just run:
 
 [source,console]
 ----
-$ cargo build --release --bin omicron-package
-$ pfexec ./target/release/omicron-package uninstall
+$ ./tools/scrimlet/softnpu-init.sh
 ----
 
-Once all the omicron services are uninstalled, you can also remove the
-previously created virtual hardware as mentioned above:
+If your external network is one you made up above, you'll need to override the gateway IP to be the address of the global zone on your made-up network.  In our example:
```

Comment:
I would probably replace this with something like the following.

The `softnpu-init.sh` script will attempt to determine a network gateway to use by looking at your operating system's default gateway. If your default gateway is not on the network you configured above (`192.168.1.0/24` in this example), then you'll need to explicitly specify what gateway to use by setting the `GATEWAY_IP` environment variable.

Reply:
Part of what I found hard to follow about the existing docs is that there are a bunch of instructions like "if <situation I can't tell if it applies to me>, then fill in <some value I'm not sure how to fill in>".  What you wrote here is definitely better because it's more explicit about the condition.  But I think it's useful to be even more explicit: "if you chose option 2 above with this network, this is exactly what you should run".

I'm inferring that maybe this is _too_ specific though?  I reworded this in da57b53515c38a699fd5e2960e24d03160fd0fcc.  Let me know what you think.